lib/clients/web/facets/chat.js

```diff
@@ -130,13 +130,19 @@ ChatFacet.prototype.update = function update(ts, channel, text, opts, optCb) {
  * @param {?} ts - Timestamp of the message to be updated.
  * @param {?} channel - Channel of the message to be updated.
  * @param {string} unfurls - a map of URLs to structured message attachments
+ * @param {Boolean} userAuthRequired  - Pass true to require user authorization.
  * @param {function=} optCb Optional callback, if not using promises.
  */
-ChatFacet.prototype.unfurl = function unfurl(ts, channel, unfurls, optCb) {
-  var requiredArgs = {
+ChatFacet.prototype.unfurl = function unfurl(ts, channel, unfurls, userAuthRequired, optCb) {
```

Comment:
i think the right way to do this would be to add an `opts` parameter, rather than explicitly putting `userAuthRequired` as a param in the signature. then you could transparently pass that through to `this.makeAPICall()` below. You can look above at the `update()` method to see an example of this pattern.